src/components/molecules/Visualizer/Widget/index.tsx

```diff
@@ -84,7 +84,9 @@ export default function WidgetComponent<PP = any, SP = any>({
     : "both";
 
   return Builtin ? (
-    <Builtin {...props} widget={w} layout={layout} extended={extended} onExtend={onExtend} />
+    <BuiltinWrapper editing={editing}>
+      <Builtin {...props} widget={w} layout={layout} extended={extended} onExtend={onExtend} />
+    </BuiltinWrapper>
```

Comment:
It does not seem to be enough to require a separate component.

```suggestion
    <div style={{ pointerEvents: editing ? "none" : "auto" }}>
      <Builtin {...props} widget={w} layout={layout} extended={extended} onExtend={onExtend} />
    </div>
```